### PR TITLE
ovirt_disk: add upload image warning for correct format

### PR DESCRIPTION
**Pull request by mnecas**
Add warning when uploading iso and the format is cow, before this PR the upload would end on FINALIZING_FAILURE and no more information, now it shows warning which will inform the user to update to raw.

@mwperina  @dangel101

### CODE REVIEW

File: plugins/modules/ovirt_disk.py [hunk 1]
```diff
@@ -731,6 +733,8 @@ def main():
 
             # Upload disk image in case it's new disk or force parameter is passed:
             if module.params['upload_image_path'] and (is_new_disk or module.params['force']):
+                if module.params['format'] == 'cow' and module.params['content_type'] == 'iso':
+                    module.warn("Parameter 'format' should not be 'cow' when uploading iso image, please change it to 'raw'.")
```
Comment (mwperina) [inline plugins/modules/ovirt_disk.py:737]: Please rephrase: `To upload an ISO image 'format' parameter needs to be set to 'raw'`

File: plugins/modules/ovirt_disk.py [hunk 2]
```diff
@@ -73,6 +74,7 @@
             - Specify format of the disk.
             - Note that this option isn't idempotent as it's not currently possible to change format of the disk via API.
         choices: ['raw', 'cow']
+        default: 'cow'
```
Comment (mwperina) [inline plugins/modules/ovirt_disk.py:77]: Does it make sense to have a default format?

Comment (mnecas) [reply plugins/modules/ovirt_disk.py:77]: Not sure.
There is already a default format [1], this just updates the documentation.
[1] https://github.com/oVirt/ovirt-ansible-collection/blob/master/plugins/modules/ovirt_disk.py#L654

Comment (mwperina) [reply plugins/modules/ovirt_disk.py:77]: OK, I missed it's already default in the code

**Pull request merged**